kmsencryption/lib.py

```diff
@@ -53,19 +55,36 @@ def decrypt(data, env, profile, prefix):
     return decrypt_value(data, prefix, kms_key_provider)
 
 
+def decrypt_object(input, prefix, kms_key_provider):
+    if not input:
+        return input
+    if isinstance(input, string_types):
+        output = decrypt_value(input, prefix, kms_key_provider) if input.startswith(prefix) else input
+        return output.decode('utf-8')
+    if isinstance(input, dict):
+        output = {}
+        for name, value in iteritems(input):
+            output[name] = decrypt_object(value, prefix, kms_key_provider)
+        return output
+    if isinstance(input, list):
+        output = []
+        for value in input:
+            output.append(decrypt_object(value, prefix, kms_key_provider))
+        return output
+    return input
+
+
 def decrypt_json(input, profile, prefix):
     kms_key_provider = get_key_provider(None, profile)
-    input_map = json.load(input)
-    output = {}
-    for name, value in input_map.iteritems():
-        output[name] = decrypt_value(value, prefix, kms_key_provider) if value.startswith(prefix) else value
+    input_object = json.load(input)
```

Comment:
Minor: ditto (`input` builtin)

Reply:
Note: Even the github is colouring that keyword. Not a harm, but my activates my OCD